crates/uv/tests/it/help.rs

```diff
@@ -469,10 +469,9 @@ fn help_subsubcommand() {
     Python versions are installed into the uv Python directory, which can be retrieved with `uv python
     dir`.
 
-    A `python` executable is not made globally available, managed Python versions are only used in uv
-    commands or in active virtual environments. There is experimental support for adding Python
-    executables to a directory on the path — use the `--preview` flag to enable this behavior and `uv
-    python dir --bin` to retrieve the target directory.
+    By default, Python executables are added to a directory on the path as `pythonX.y`. To install
+    `python3` and `python`, use the `--default` flag. Use `uv python dir --bin` to retrieve the target
```

Comment:
(shared with the other comment) Should we document that --default is still in preview?

Reply:
I considered it. I'm not sure if it's important and was leaning towards not.

Reply:
I'll add it.